ftests/ftests_test.go

```diff
@@ -521,7 +533,9 @@ func (c *Host) Share(url string) error {
 		w := writeFunc(func(p []byte) (int, error) {
 			b, err := ansi.Strip(p)
 			if err != nil {
-				return 0, err
+				// Ignore ANSI parsing errors (e.g., malformed OSC sequences on Windows)
+				// and use the original bytes instead
+				b = p
```

Comment:
Perhaps adding a warn logging for visibility:

```suggestion
				testLogger.Warn("failed to strip ANSI codes", "error", err)
				b = p
```